frontend/src/Commons/Forms/InputModules/FarmInformation/FarmInformation.style.ts

```diff
@@ -5,14 +5,14 @@
  */
 import styled from '@emotion/styled';
 
-const StyledFarmInfo = styled.form`
+const StyledFarmInfo = styled.div`
   display: flex;
   flex-direction: column;
   justify-content: space-between;
   align-items: left;
   height: 100%;
   p {
-    margin-bottom: 0;
+    margin: 8px 0 0 0;
```

Comment:
this is easily read as margin-top: 8px;

Reply:
P tags come with some predefined styling top and bottom

Reply:
The thing is this is removing the margin bottom, which I don't want for the p tag because it's adding too much spacing between it and their corresponding input field.

Reply:
I see gotcha.